tools/cli/commands.go

```diff
@@ -187,6 +189,7 @@ func RegisterDomain(c *cli.Context) {
 	description := c.String(FlagDescription)
 	ownerEmail := c.String(FlagOwnerEmail)
 	retentionDays := defaultDomainRetentionDays
+	securityToken := c.String(FlagSecurityToken)
```

Comment:
nitpick can we move this code 3 lines below

Reply:
OK